src/components/Tool/Home/Home.tsx

```diff
@@ -56,6 +40,9 @@ const variants = {
 const ToolHome = (): ReactElement => {
```

Comment:
It doesn't matter much, but I'd probably lean towards making `onSetupTour` a prop so that it's calculated in once place.